### PR TITLE
Add documentation on how to run a Hackathon

### PR DESCRIPTION
**Pull request by jasonblais**
@amyblais requesting for editor review (grammar, formatting, etc)

@enahum requesting for dev review. High-level overview is sufficient. Note that I incorporated some of the dev feedback into the process, and added the rest as suggestions to consider for future.

@adamjclarkson requesting for PM review. High-level overview is sufficient. I thought to ask you for high-level review to see if things make sense from someone's perspective who is the newest member of the PM team.

### CODE REVIEW

File: source/process/community-guidelines.rst [hunk 1]
```diff
@@ -95,6 +95,11 @@ Tips
 4. Consider sequencing design review meetings with design priorities.
 5. Encourage less vocal attendees to share feedback via written chat.
 
+How to run a hackathon?
```
Comment (amyblais) [inline source/process/community-guidelines.rst]:
```suggestion
How to run a Hackathon?
```

File: source/process/how-to-run-a-hackathon.rst [hunk 2]
```diff
@@ -0,0 +1,114 @@
+How to run a Hackathon
+=========================
+
+This page helps outline best practices and recommended steps for running a Hackathon.
```
Comment (amyblais) [inline source/process/how-to-run-a-hackathon.rst]:
```suggestion
This page outlines best practices and recommended steps for running a Hackathon.
```

File: source/process/how-to-run-a-hackathon.rst [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+Previously, Hackathons have run twice a year with one coinciding with MatterCon. In future, Hackathons may be run 3 or 4 times a year.
+
+T-minus counts measure "working days" (weekdays other than major holidays concurrent in US and Canada) prior to the Hackathon day. All times are based on 8:00am ([Palo Alto Time](http://everytimezone.com/)) unless otherwise stated.
```
Comment (amyblais) [inline source/process/how-to-run-a-hackathon.rst]: Wrong link formatting.

File: source/process/how-to-run-a-hackathon.rst [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+A. (T-minus 60 working days)
+----------------------------------------------------------------
+
+1. Decide dates for the Hackathon. Some things to take into consideration:
+
+  - Avoid overlap with major holidays in US, Canada, Europe or Asia (including Australia and Japan).
+  - Avoid overlap with significant release dates, such as release testing or feature complete. Recommended time is starting a Hackathon within a week after major feature complete.
```
Comment (amyblais) [inline source/process/how-to-run-a-hackathon.rst]:
```suggestion
  - Avoid overlap with significant release dates, such as release testing or feature complete. Recommended time is starting a Hackathon within a week after feature complete.
```

File: source/process/how-to-run-a-hackathon.rst [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+
+1. Put together a project plan, including task description, start date, due date, owner and any additional notes. `See an example <https://docs.google.com/spreadsheets/d/15iiLPzJjlajHQVgQa9HulW9WfBauIlnkCOJ07oKz7_U>`_.
+2. Identify 1 or 2 members in marketing to partner with, who can help with tweets, blog posts and other marketing content to generate awareness of the Hackathon.
+3. Identify 1 or 2 members in ops to partner with, who can help with meeting scheduling, SWAG and other operational tasks.
+
+C. (T-minus 30 working days)
+----------------------------------------------------------------
+
+1. Work with ops to decide on what SWAG to send for participation - both for contributors and staff.
+2. Create a landing page for the Hackathon, which contains all information and to which other communication links to.
```
Comment (amyblais) [inline source/process/how-to-run-a-hackathon.rst]:
```suggestion
2. Create a landing page for the Hackathon, which contains all information and to which other communication items are linked to.
```

File: source/process/how-to-run-a-hackathon.rst [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+  - All processes and documentations should be clearly included on this landing page. This should include process for finding and assigning teams, and submission guidelines. FAQs can be incorporated as needed.
+
+3. Create a GitHub repository for submitting Hackathon projects. This can be the same as the landing page if a GitHub repository is used for it.
+
+D. (T-minus 20 working days)
+----------------------------------------------------------------
+
+1. Announce the Hackathon internally by posting in `Private Staff Discussion channel <https://community.mattermost.com/private-core/channels/platform>`_ and sharing on R&D weekly meeting.
+2. Schedule a kick-off call with all Hackathon stakeholders, including the developer, marketing and ops members identified previously.
+3. Define and set up metrics to measure impact of the Hackathon. Examples include: number of participants (staff + community separately), number of submissions, number of submissions shipped as a core feature, or as a plugin in the marketplace.
```
Comment (amyblais) [inline source/process/how-to-run-a-hackathon.rst]:
```suggestion
3. Define and set up metrics to measure impact of the Hackathon. Examples include: number of participants (staff + community separately), number of submissions, number of submissions shipped as a core feature or as a plugin in the marketplace.
```

File: source/process/how-to-run-a-hackathon.rst [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+
+E. (T-minus 15 working days)
+----------------------------------------------------------------
+
+1. Create Meetup links for Hackathon idea presentation, kick-off and demos. For examples, see `Mattermost Meetup page <https://www.meetup.com/mattermost/>`_.
+2. Share the Hackathon externally via Reddit, GitHub, Discourse and the `community server <https://community.mattermost.com>`_. For examples, see previous `Reddit post <https://www.reddit.com/r/Mattermost/comments/dvvsm4/mattermost_hackathon_nov_22_26/>`_, and `GitHub issue <https://github.com/mattermost/mattermost-server/issues/13087>`_.
+3. Publish a blog post announcing the Hackathon to the Mattermost community, with links back to landing page for details on how to participate.
```
Comment (amyblais) [inline source/process/how-to-run-a-hackathon.rst]:
```suggestion
3. Publish a blog post announcing the Hackathon to the Mattermost community, which links back to the landing page for details on how to participate.
```

File: source/process/how-to-run-a-hackathon.rst [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+F. (T-minus 10 working days)
+----------------------------------------------------------------
+
+1. Schedule a series of tweets announcing and reminding community of the upcoming Hackathon. See `this Google Doc <https://docs.google.com/document/d/1W5j5dYa3ZjSgEocniWXTEO2xjj5A8S9ihD03snUnLnA/edit#heading=h.s3nujhxhjrxo>`_ for examples.
+2. Work with Ops team to get SWAG prizes ready and set up with Printfection.
+
+G. (T-minus 5 working days)
+----------------------------------------------------------------
+
+1. Hold a Hackathon idea presentation call, scheduled for 60 minutes, with an additional 30 minutes for extra time. This time is used to gather feedback about ideas and attract others to work as a team. To see previous slides and meeting recording, see `Hackathon Nov/2019 Google Drive <https://drive.google.com/drive/folders/1iBVGDzdwZ72oEY6mshzjcgJQ4NdhXRfW>`_.
```
Comment (amyblais) [inline source/process/how-to-run-a-hackathon.rst]:
```suggestion
1. Hold a Hackathon idea presentation call, scheduled for 60 minutes, with an additional 30 minutes for extra time. This time is used to gather feedback about ideas and to attract others to work as a team. To see previous slides and meeting recording, see `Hackathon Nov/2019 Google Drive <https://drive.google.com/drive/folders/1iBVGDzdwZ72oEY6mshzjcgJQ4NdhXRfW>`_.
```

File: source/process/how-to-run-a-hackathon.rst [hunk 9]
```diff
@@ -0,0 +1,114 @@
...
+1. Hold a Hackathon kick-off call, scheduled for 30 minutes. To see previous slides and meeting recording, see `Hackathon Nov/2019 Google Drive <https://drive.google.com/drive/folders/1iBVGDzdwZ72oEY6mshzjcgJQ4NdhXRfW>`_.
+
+I. (T-plus 3 working days)
+----------------------------------------------------------------
+
+1. Hold a Hackathon demo call, scheduled for 2 hours, with an additional hour for extra time. This time is used for participants to present their project. Invite the entire Mattermost company to the call as optional attendee.
+
+  - Provide guidelines and best practices for creating presentations (both for staff and community).
+  - Three minutes allowed per Hackathon participant. For multiple submissions, five minutes total is allowed to present.
+  - Ask participants who can't attend submit a demo video, which are shared during the call.
```
Comment (amyblais) [inline source/process/how-to-run-a-hackathon.rst]:
```suggestion
  - Ask participants who can't attend to submit a demo video, which are shared during the call.
```

File: source/process/how-to-run-a-hackathon.rst [hunk 10]
```diff
@@ -0,0 +1,114 @@
...
+----------------------------------------------------------------
+
+1. Publish blog post sharing insights of the Hackathon.
+2. Update this process doc for changes in running a Hackathon.
+
+Tips and Best Practices
+----------------------------------------------------------------
+
+1. Do not use your own Zoom link for Hackathon calls. Recommend using a company Zoom link, or one created specifically for Hackathon.
+2. In Zoom, go to **Settings > Recording** and set **Recording consent** to true. This prompts participants for consent to be recorded when recording starts.
```
Comment (amyblais) [inline source/process/how-to-run-a-hackathon.rst]:
```suggestion
2. In Zoom, go to **Settings > Recording** and set **Recording consent** to true. This prompts participants to consent to be recorded when recording starts.
```

**Comment by jasonblais** [comment]
@amyblais editor feedback addressed via https://github.com/mattermost/docs/pull/3202/commits/70f0458442f4b4991487134c13ef6fe81815b080

@adamjclarkson Good question - and I'd say it's a potential opportunity to consider in future. There was some interest from at least the support team to be more involved in the Hackathon.

**Pull request merged**